c/models/column_hasher.cc

```diff
@@ -20,17 +20,18 @@
 // IN THE SOFTWARE.
 //------------------------------------------------------------------------------
 #include "models/column_hasher.h"
+#include <float.h>
```

Comment:
I would avoid including any non-standard headers unless absolutely necessary.
Plus, it would be better to define `DBL_MANT_DIG` as an explicit constexpr anyways: at least then there is no doubt about its value (which should be 52).

Reply:
If we are sure that on all the platforms, where datatable runs, `DBL_MANT_DIG == 52` then let me define it somewhere in ftrl part.

Reply:
Btw, I'm not sure `float.h` is not standard, it is just an [old name](https://en.cppreference.com/w/cpp/header/cfloat) for the standard `<cfloat>`.

Reply:
This is part of IEEE754 standard for doubles (https://en.wikipedia.org/wiki/Double-precision_floating-point_format). This is true on all platforms where doubles use IEEE754 format; and if they don't, then this approach would not be correct anyways.
Also note that DBL_MANT_DIG is defined as 53, which is not what we want.

Reply:
Yes, I noticed that too, not sure why it is 53 :) Anyways, I've already defined it in dt::Ftrl class.